### PR TITLE
Givermei devsecops refresh

### PR DESCRIPTION
**Pull request by vermegi**
@wviriya please review. 
I didn't add a threat analysis example (yet).

### CODE REVIEW

File: content/devops/devsecops/Operate.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Operate
+
+Once you start operating your workloads, you need a means to continuously assess, secure and defend them.
+
+![Defender for Cloud synopsis](media/defender-for-cloud-synopsis.png)
+
+This is where [Microsoft Defender for Cloud](https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-cloud-introduction) steps in. Here as well there is a dedicated [FTA Live session](https://github.com/Azure/FTALive-Sessions/blob/main/content/security/azure-security/AzureSecurityCenter.MD) for which you can sign up [here](https://fasttrack.azure.com/live/category/Security).
```
Comment (wviriya) [inline content/devops/devsecops/Operate.md:7]:
```suggestion
This is where [Microsoft Defender for Cloud](https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-cloud-introduction) steps in. Here as well, there is a dedicated [FTA Live session](https://github.com/Azure/FTALive-Sessions/blob/main/content/security/azure-security/AzureSecurityCenter.MD) for which you can sign up [here](https://fasttrack.azure.com/live/category/Security).
```

**Review by wviriya — COMMENTED** [review]
A minor grammar correction, otherwise, this looks great. My only concern is time, now that you add quite a bit of content.

File: content/devops/devsecops/README.md [hunk 2]
```diff
@@ -20,14 +20,32 @@ There are many secure software development frameworks being used out in the worl
 1. [Microsoft Secure Development Lifecycle](https://www.microsoft.com/en-us/securityengineering/sdl/practices).
 1. [SafeCode fundamental practices for Secure Software Development](https://safecode.org/fundamental-practices-secure-software-development-2/).
```
Comment (wviriya) [inline content/devops/devsecops/README.md:21]:
```suggestion
1. [SafeCode fundamental practices for Secure Software Development](https://safecode.org/uncategorized/fundamental-practices-secure-software-development).
```

**Comment by vermegi** [comment]
Yeah, timing might now indeed be a concern, I agree. I will do a dry-run for myself today to see where I end up with timing. We can still decide to make customers in the session decide what we talk about in the session so we only pick up the points that are most relevant to them ... At least, that was my thinking on that.

**Comment by vermegi** [comment]
Added some link refreshes. Can someone approve so PR can be merged?

**Review by vanessabruwer — APPROVED** [review]

**Pull request merged**